docs/game_data/spel2.lua

```diff
@@ -64,8 +64,8 @@ function F(f_string) end
 
 ---Returns Player (or PlayerGhost if `get_player(1, true)`) with this player slot
 ---@param slot integer
----@param or_ghost boolean
----@return Player
+---@param or_ghost boolean?
+---@return any
 function get_player(slot, or_ghost) end
```

Comment:
I don't like this turning to `any`, can I push a commit here to make it return `Player|Playerghost`?

Reply:
Sure

Reply:
actually you only would have to add `// lua["get_player"] = [&lua](int8_t slot, std::optional<bool> or_ghost) -> Player|PlayerGhost` before the actual `"get_player"` for this